aspnetcore-webapi/RateLimitingDemo/RateLimitingDemo.UsingCustomMiddleware/Repositories/ProductCatalogRepository.cs

```diff
@@ -0,0 +1,57 @@
+﻿using RateLimitingDemo.UsingCustomMiddleware.Model;
+
+namespace RateLimitingDemo.UsingCustomMiddleware.Repositories;
+public class ProductCatalogRepository : IProductCatalogRepository
```

Comment:
Same thing, move it to the shared class library.

Reply:
done